README.md

```diff
@@ -79,13 +79,9 @@ The project is split in several Maven modules:
 
 * _engine_: The engine of the project. Most of the beef is here.
 
-* _infinispan_: Backend storing indexes in [Infinispan](http://infinispan.org/).
-
 * _integrationtest_: Integration tests with various technologies like WildFly, Spring or Karaf.
 Also includes performance tests.
 
-* _legacy_: Old Maven GAV kept for backward compatibility.
```

Comment:
I'd prefer to keep it around for longer, especially as still recently some documentation was advising those GAVs.. but since it's now using an explicit "relocation" tag I didn't feel like needing to scare first time contributors who are reading the README with a long list of modules, when these can be ignored.